buildSrc/src/main/kotlin/jneedle.publishing.gradle.kts

```diff
@@ -0,0 +1,94 @@
+plugins {
+    signing
+    `maven-publish`
+}
+
+publishing {
+    publications {
+        val projectName = project.fullName.split("-").joinToString(separator = " ") { word -> word.capitalize() }
+        val projectGroup = project.groupString
+        val projectVersion = project.versionString
+        val projectDescription = """
+            Yet another jar malware detection tool
+        """.trimIndent()
+        val projectUrl = Repository.projectUrl
+        val projectBaseUri = Repository.projectBaseUri
+
+        val licenseName = "MIT"
+        val licenseUrl = "https://mit-license.org/"
+
+
+        create<MavenPublication>("maven") {
+            groupId = projectGroup
+            artifactId = project.fullName
+            version = projectVersion
+            from(components["java"])
+
+            pom {
+                name.set(projectName)
+                description.set(projectDescription)
+                url.set(projectUrl)
+
+                inceptionYear.set("2023")
+
+                licenses {
+                    license {
+                        name.set(licenseName)
+                        url.set(licenseUrl)
+                    }
+                }
+                developers {
+                    developer {
+                        id.set("kosmx")
+                        name.set("KosmX")
+                        email.set("kosmx.mc@gmail.com")
+                    }
+
+                    developer {
```

Comment:
Ok, this should be moved to a contributors list file, but that can wait

Reply:
tbh it kinda *is* in its own file already lol